### ISSUE TITLE
How to put class pixel weight in Unet_Medical example

### ISSUE THREAD

**Issue by yaozr** [NVIDIA/DeepLearningExamples#263]
I'm doing binary class segmentation of images, and I have the sample image where the target class only occupy around 4% of image pixel and all other are labelled as background, the result is it predict every image as background. How should I put this weight information into the training framework? Many thanks for tips.

**Comment by pribalta** [comment]
@yaozr your problems seems to be related to a class imbalance in your data. It is going to be difficult if only 4% of the pixels correspond to your target class, but you can experiment by using class weights when you compute the loss during training.

The idea would be to take crossentropy and dice losses [here](https://github.com/NVIDIA/DeepLearningExamples/blob/master/TensorFlow/Segmentation/UNet_Medical/utils/runner.py#L137:L140), and multiply them by a [1, N] vector, where N is you number of classes.

This vector should contain the class weights, meaning that a vector of e.g. [0.05, 0.95] weights will put more emphasis on the loss for the second class. You can experiment with different weight coefficients and attempt to mitigate the imbalance problems.

**Comment by yaozr** [comment]
Many thanks!
